src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -921,7 +926,10 @@ private bool CheckParameterValueKind(SyntaxNode node, BoundParameter parameter,
 
             return true;
         }
+    }
 
+    internal partial class RefSafetyAnalysis
```

Comment:
Do you plan to submit a follow-up PR to consolidate these partial types?

Reply:
I will merge the `partial class RefSafetyAnalysis` parts into a single file in a subsequent PR that only moves code.